apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/lru/EvictionPolicy.java

```diff
@@ -0,0 +1,97 @@
+package com.apollographql.android.cache.normalized.lru;
+
+import com.apollographql.android.api.graphql.internal.Optional;
+
+import java.util.concurrent.TimeUnit;
+
+public final class EvictionPolicy {
+
+  public static final EvictionPolicy NO_EVICTION = EvictionPolicy.builder().build();
+
+  private final Optional<Long> maxSizeBytes;
+  private final Optional<Long> maxEntries;
+  private final Optional<Long> expireAfterAccess;
+  private final Optional<TimeUnit> expireAfterAccessTimeUnit;
+  private final Optional<Long> expireAfterWrite;
+  private final Optional<TimeUnit> expireAfterWriteTimeUnit;
+
+  public Optional<Long> maxSizeBytes() {
```

Comment:
Good point, I think this is actually just for internal use, after building the policy and passing to the cache, you shouldn't need to access the settings. will change to package private.